Documentation/warnings.rst

```diff
@@ -0,0 +1,30 @@
+Cilium Warnings
+###############
+
+..
+   NOTES: each warning has an id (e.g., cgrouplowproccount) used in the code to
+   refer to specifics in this document. Code points to the latest version of the
+   docs, so text should be valid for all supported Cilium versions.
```

Comment:
How do you plan to update the document if the exact meaning of a warning change between several versions? Why not make the links point to the version of the doc that corresponds to the running version? (I think `version.GetCiliumVersion()` can be used to get the version?)